velox/functions/prestosql/aggregates/tests/AggregationTestBase.cpp

```diff
@@ -221,6 +221,38 @@ void AggregationTestBase::testAggregations(
     }
   }
 
+  if (!groupingKeys.empty() && allowInputShuffle_) {
+    SCOPED_TRACE("Run partial + final with abandon partial agg");
+    PlanBuilder builder(pool());
+    makeSource(builder);
+
+    core::PlanNodeId partialNodeId;
+    builder.partialAggregation(groupingKeys, aggregates)
+        .capturePlanNodeId(partialNodeId)
+        .finalAggregation();
+
+    if (!postAggregationProjections.empty()) {
+      builder.project(postAggregationProjections);
+    }
+
+    AssertQueryBuilder queryBuilder(builder.planNode(), duckDbQueryRunner_);
+    queryBuilder.config(core::QueryConfig::kAbandonPartialAggregation, "true")
```

Comment:
Should we also add a test that verifies that partial agg gets disabled when kAbandonPartialAggregation is false?